### ISSUE TITLE
Recieve notifications don't alert unless app is in foreground 

### ISSUE THREAD

**Issue by pinheadmz** [voisine/breadwallet#379]
When Bitcoin is sent to breadwallet, it does not notify user unless the app is open and in front. Even the badges I noticed don't appear unless the app is switched to the front, then the badge immediately clears because the app is open! 

I'm not sure if bread is able to remain synced with the network in the background. My experience is that no, it can't update the blockchain unless you are looking at the app, even if it's "still open" in the background. 

I would love receive alerts, even with sounds and lock screen notifications if possible.

**Comment by voisine** [comment]
The app will do background network fetch every few hours unless you manually kill it. The app badge will be updated then along with a lock screen notification if you have it turned on in settings.

In the future we might offer a service where the app can register a bloomfilter, and then receive push notifications whenever a tx that matches the filter happens. The app would then discard any false positive notifications. I'll leave this open as a feature request.

**Comment by pinheadmz** [comment]
Oh! "Every few hours!" 
I think I'm spoiled by instant notifications from centralized server based wallets and my full node... Any way to change the frequency of the refresh?

**Comment by voisine** [comment]
Every few hours is as often as iOS will do background network fetches, so that's the best we can do with a pure client side solution, short of doing some evil battery draining hack like continuously streaming a silent audio file in the background.

**Comment by dabura667** [comment]
Yeah, this is unfortunately the limtation of not having a centralized server to push notifications to you.
